### PR TITLE
Adds black owned banner to partner page

### PR DESCRIPTION
**Pull request by lilyfromseattle**
This is a followup to [GRO-820](https://artsyproduct.atlassian.net/browse/GRO-820). Design decided that a static banner was preferable to [an animated marquee](https://github.com/artsy/eigen/pull/6305) so I reimplemented the feature here.

Screen Shot 2022-04-05 at 12 43 23 PM

### CODE REVIEW

File: src/app/Scenes/Partner/Components/PartnerHeader.tsx [hunk 1]
```diff
@@ -10,27 +11,31 @@ const PartnerHeader: React.FC<{
   partner: PartnerHeader_partner
 }> = ({ partner }) => {
   const eligibleArtworks = partner.counts?.eligibleArtworks ?? 0
+  const isBlackOwned = partner.categories!.filter((c) => c && c.name === "Black Owned").length > 0
```
Comment (jonallured) [inline src/app/Scenes/Partner/Components/PartnerHeader.tsx:14]: Might be worth copy/pasting what @The-Beez-Kneez did to support woman owned here too?

https://github.com/artsy/force/pull/9771/files#diff-dca59a5c950ae568a2ff6901b82b16b30272e9bf08c393d04b541e1244d459fd

Comment (The-Beez-Kneez) [reply src/app/Scenes/Partner/Components/PartnerHeader.tsx:14]: Second this!

**Review by The-Beez-Kneez — APPROVED** [review]
Nit pick: We could add a test for Women Owned as well. But the functionality is tested with the Black Owned test, so I am fine skipping it.

**Review by jonallured — APPROVED** [review]
LGTM!! 🎉

**Pull request merged**